### PR TITLE
[Fixes #3684] Keep original projection and BBOX / general cleanup of resource models and logs

### LINKED ISSUE #3684: Keep original projection and BBOX
Stores on GeoNode the original layer projection and BBOX instead of LatLon one by default.

### PR DESCRIPTION
**Pull request by afabiani**
- [x] Fixes #3684 : Keep original projection and BBOX 
 - [x] General cleanup of resource models and logs

### CODE REVIEW

File: geonode/client/templates/geoext/layers/layer_geoext_map.html [hunk 1]
```diff
@@ -147,38 +147,35 @@
                         var map = app.mapPanel.map;
                         var layer = app.map.layers.slice(-1)[0];
                         var bbox = layer.bbox;
-                        var crs = layer.crs
+                        var crs = layer.srs
```
Comment (capooti) [inline geonode/client/templates/geoext/layers/layer_geoext_map.html]: a semicolon is missing here

File: geonode/client/templates/geoext/layers/layer_geoext_map_mini.html [hunk 2]
```diff
@@ -141,38 +141,35 @@
                         var map = app.mapPanel.map;
                         var layer = app.map.layers.slice(-1)[0];
                         var bbox = layer.bbox;
-                        var crs = layer.crs
+                        var crs = layer.srs
```
Comment (capooti) [inline geonode/client/templates/geoext/layers/layer_geoext_map_mini.html]: same here

File: geonode/client/templates/geoext/maps/map_geoexplorer.js [hunk 3]
```diff
@@ -159,39 +159,35 @@ Ext.onReady(function() {
                 var map = app.mapPanel.map;
                 var layer = app.map.layers.slice(-1)[0];
                 var bbox = layer.bbox;
-                var crs = layer.crs
-
+                var crs = layer.srs
```
Comment (capooti) [inline geonode/client/templates/geoext/maps/map_geoexplorer.js]: same here

File: geonode/geoserver/signals.py [hunk 4]
```diff
@@ -223,19 +224,20 @@ def geoserver_post_save_local(instance, *args, **kwargs):
...
     instance.store = gs_resource.store.name
 
-    bbox = gs_resource.latlon_bbox
-
-    # FIXME(Ariel): Correct srid setting below
-    # self.srid = gs_resource.src
-
-    instance.srid_url = "http://www.spatialreference.org/ref/" + \
-        instance.srid.replace(':', '/').lower() + "/"
+    bbox = gs_resource.native_bbox
```
Comment (capooti) [inline geonode/geoserver/signals.py]: maybe check if bbox is not None

Comment (capooti) [reply geonode/geoserver/signals.py]: this can apply also to other parts. Can we assume native_bbox is always populated?

**Comment by afabiani** [comment]
Thanks @capooti will apply your fixes before proceeding with 2.8

File: geonode/geoserver/signals.py (hunk 4 above)
Comment (afabiani) [reply geonode/geoserver/signals.py]: by using gsconfig, yes (at least with current version).

**Pull request merged**